customisation/alignment.md

```diff
@@ -1,6 +1,27 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Alignment
-description:
+description: Linking corresponding sentences in the input and output languages
 ---
+
+**Alignment** is the process of identifying and linking the corresponding sentences in the input and output languages.
+
+Alignment can be used to create [parallel data](/parallel-data).
+The aligned parallel corpora are then used to train machine translation models.
```

Comment:
Could we link the term "train" to [`training`](/training), even though it doesn't exist yet?